src/components/video/index.jsx

```diff
@@ -18,21 +19,30 @@ const Video = ({ className, aspectRatio, viewportPercentage, embedMarkup, ...res
 
 	const containerClassNames = [COMPONENT_CLASS_NAME, className].filter((i) => i).join(" ");
 
+	const truncate = (num) => Math.trunc(num * 1000) / 1000;
```

Comment:
```suggestion
	const truncate = (num) => Math.trunc(num * 10000) / 10000;
```
Let's do 4 digits for 16/9 parity.  I know I said 3 previous, but I think 4 would be "safe"